app/controllers/index.rb

```diff
@@ -1,21 +1,44 @@
 require 'sinatra'
 
 get '/' do
-  <<-HTML
-<p>
-Welcome to my LightSaber.  Prepare to be amazed.
-<p>
-<a href='/sound'>Hear my swooshy sound!</a>
-  HTML
+  erb :index
 end
 
-get '/sound' do
-  <<-HTML
-<audio controls>
-  <source src="http://soundbible.com/grab.php?id=19&type=wav" type="audio/wav">
-  <source src="http://soundbible.com/grab.php?id=19&type=mp3" type="audio/mpeg">
-Your browser does not support the audio element.
-</audio>
-  HTML
+#   <<-HTML
+# <p>
+# Welcome to my LightSaber.  Prepare to be amazed.
+# <p>
+# <a href='/sound'>Hear my swooshy sound!</a>
+#   HTML
+# end
+
+get '/users/:name' do
+	@name = params[:name]
+	erb :user_info
+end
+
+
+
+# get '/sound' do
+#   <<-HTML
+# <audio controls>
+#   <source src="http://soundbible.com/grab.php?id=19&type=wav" type="audio/wav">
+#   <source src="http://soundbible.com/grab.php?id=19&type=mp3" type="audio/mpeg">
+# Your browser does not support the audio element.
+# </audio>
+#   HTML
+# end
+
+post '/users/new' do
+	new_user = User.new(name: params[:name], password: params[:password])
+	redirect "/users/#{new_user.name}"
```

Comment:
This should redirect to /users/#{new_user.id}

Dont redirect to name - it may not be unique in your database.  ID is always unique